Gaijinizer patch/Map172.txt

```diff
@@ -381,41 +390,44 @@ person as it is.#MTLed
 ミカを殺した時ですら、全て『ユウマ』の本心じゃ
 > CONTEXT: Map172/events/2/pages/0/204/Dialogue/enemy17
 > CONTEXT: Map172/events/2/pages/1/377/Dialogue/enemy17
-\\C[2]Words of concern for you...Encouragement ...
-Anhui...Even when you killed Micah, it's all
-about Yuuma#MTLed
+\\C[2]Those times I worried about you, encouraged you,
+reassured you, even when I killed Micah, those
+were all "Yuuma's" true feelings.#Ron
 > END STRING
```

Comment:
minor flow:
>_**Those times I worried about you... encouraged
you... reassured you... Even when I killed Micah,
those were all "Yuuma's" true feelings.**_